docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -275,7 +275,7 @@ Basic support has a monthly fee starting from $200.
 
 **Can I purchase Temporal Cloud through my Amazon, Azure, or Google Cloud Platform marketplace?**
 
-You can purchase Temporal Cloud credits in the AWS Marketplace.
+You can purchase Temporal Cloud credits in the AWS Marketplace. GCP marketplace purchase coming soon.
```

Comment:
every sentence on one line 😊

Reply:
Talking about upcoming changes is inherently fragile, especially if a ticket hasn't been filed to address things when a feature is released. Checked with Brian and we generally do not include.

Reply:
I'll remove it.

Reply:
Changing to `AWS Marketplace only.` as the question above does specifically call out GCP.